Sources/SwiftGenKit/Stencil/YamlParser+Context.swift

```diff
@@ -26,14 +26,7 @@ extension Yaml.Parser {
     [
       "name": file.name,
       "path": file.path.string,
-      "documents": file.documents.map(map(document:))
-    ]
-  }
-
-  private func map(document: Any) -> [String: Any] {
-    [
-      "data": document,
-      "metadata": Metadata.generate(for: document)
+      "documents": file.documents.map(StencilContextLazyDocument.init)
```

Comment:
Just had a thought: do you think it would be useful to have a `"document"` key that would alias to `documents.first`? Because sure YAMLs can technically have multiple documents in it as per the YAML specification… but let's be honest, most YAML files only contain one document and many people don't know about the ability for YAML to contain multiple docs, so… 🤔

Reply:
This is for the template writers, they have to handle input correctly. We're deprecating/removing the multiple documents from plist/json, because it isn't correct there. I'd say adding `document` to yaml would be just as incorrect.

I'd rather have correct contexts that force users to deal with the real/actual structure of things, instead of shortcuts that, in this case, barely impact context/template complexity.